changelog.txt

```diff
@@ -0,0 +1 @@
+feature - Added "firebase experimental:functions:shell" for emulating and invoking functions via an interactive shell. Instructions for usage at https://firebase.google.com/docs/functions/local-emulator.

```

Comment:
Suggested tweaks:

    feature - Added `experimental:functions:shell` command for emulating and invoking functions via an interactive shell. Instructions for usage at https://goo.gl/arxNr5